lib_nbgl/doc/nbgl_use_case.dox

```diff
@@ -25,258 +25,124 @@ for end-users, getting more and more familiar with the user experience of applic
-                         NULL, // will use the default subTagline ("This app relies on\n<masterAppName>")
-                         false, // use "info" type top-right icon
-                         myAppSettings,
-                         onQuit);
-}
-@endcode
-
-@subsection use_case_settings Settings Use Case
-
-\image{inline} html UseCase-Settings1.png "caption" height=300
-
-Usually settings (or information) of an application consist in a list of pages, each page containing:
-
-- some radio buttons to make a choice
-- some switches for various configuration
-- a list of information organized by pairs [info_name, info_value]
-
-The API to initiate the display of the series of pages is @ref nbgl_useCaseSettings(), providing:
-
-- the title of the settings series
-- the page in which to start
-- the number of pages
-- a boolean to indicate whether the title is touchable or not (if touchable, it is used to quit the series,
-so no Quit button in navigation bar)
-- a callback called when the quit button (or touchable title) is touched
-- a navigation callback called when navigation arrow buttons are touched (and to fill the initial page), to retrieve
-the content of the page
-- a last callback called when any of the controls provided in content is touched
-
-@note All tokens used for the given "settings" must be greater or equal than \ref FIRST_USER_TOKEN
+extern const nbgl_icon_details_t *eth_icon;
 
-\image{inline} html UseCase-Settings2.png "caption" height=300
-
-Here is the source code to display the first example of settings:
-
-@code
 enum {
-  SWITCH1_TOKEN = FIRST_USER_TOKEN,
-  SWITCH2_TOKEN,
-  SWITCH3_TOKEN
+    SWITCH1_TOKEN = FIRST_USER_TOKEN,
+    SWITCH2_TOKEN
 };
 
 static const nbgl_layoutSwitch_t switches[] = {
-  {
-    .initState = false,
-    .text = "Blind signing",
-    .subText = "Enable transaction blind\n" "signing",
-    .token = SWITCH1_TOKEN,
-    .tuneId = TUNE_TAP_CASUAL
-  },
-  {
-    .initState = true,
-    .text = "Debug",
-    .subText = "Display contract data details",
-    .token = SWITCH2_TOKEN,
-    .tuneId = TUNE_TAP_CASUAL
-  },
-  {
-    .initState = true,
-    .text = "Nonce",
-    .subText = "Display account nonce\n" "in transaction",
-    .token = SWITCH3_TOKEN,
-    .tuneId = TUNE_TAP_CASUAL
-  },
+    {.initState = false,
+     .text      = "ENS addresses",
+     .subText   = "Displays the resolved address of ENS domains.",
+     .token     = SWITCH1_TOKEN,
+     .tuneId    = TUNE_TAP_CASUAL},
+    {.initState = true,
+     .text      = "Raw messages",
+     .subText   = "Displays raw content from EIP712 messages.",
+     .token     = SWITCH2_TOKEN,
+     .tuneId    = TUNE_TAP_CASUAL}
 };
+static const char *infoTypes[]    = {"Version", "Developer"};
+static const char *infoContents[] = {"1.9.18", "Ledger"};
 
-static const char* infoTypes[] = {
-  "Version"
-};
+// function called in case of action on switches
+static void controlsCallback(int token, uint8_t index, int page);
```

Comment:
Why don't we keep a code sample of this function content?

Reply:
Done :)